patches/chromium/pepper_plugin_support.patch

```diff
@@ -1,9 +1,10 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: deepak1556 <hop2deep@gmail.com>
-Date: Thu, 20 Sep 2018 17:48:09 -0700
-Subject: pepper_flash.patch
+Date: Thu, 20 Sep 2018 17:46:17 -0700
+Subject: pepper plugin support
 
-Allows building chrome pepper flash integration for Electron.
+This tweaks Chrome's pepper flash and PDF plugin support to make it
```

Comment:
Since the flash sources are now built as `source_set`, the component build specific patch is not needed

`class COMPONENT_EXPORT(PEPPER_FLASH) ChromeBrowserPepperHostFactory : public ppapi::host::HostFactory` => `class ChromeBrowserPepperHostFactory : public ppapi::host::HostFactory`

Reply:
done